app/config/AppConfig.scala

```diff
@@ -78,6 +78,11 @@ class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesCon
     else etmpBase + s"/penalty/financial-data/VRN/$vrn/VATC"
   }
 
+  def getPenaltyDetailsUrlv3(vrn: String): String = {
+    if(!isEnabled(CallAPI1812ETMP)) stubBase + s"/penalties-stub/penalties/penalty-details/VAT/VRN/$vrn"
+    else etmpBase + s"/penalties/penalty-details/VAT/VRN/$vrn"
+  }
+
```

Comment:
Could you not use the getPenaltyDetailsUrl method?

Reply:
Most likely can be deleted if changed in the connector

Reply:
> Could you not use the getPenaltyDetailsUrl method?

this method returns url '/penalty/details/VATC/VRN/' used in getPenaltyDetails method.

Reply:
But the connector call shouldn't be different to what we use?